### ISSUE TITLE
MIssing ONIX codes in tehcniques

### ISSUE THREAD

**Issue by gregoriopellegrino** [w3c/publ-a11y#696]
I'm including here the report that Vivian Luu (Booknet Canada) sent via email, thanks Vivian!

It's a list of ONIX codes that are in codelist 196, but are not present in the ONIX techniques:

- code 05 of codelist 196 (PDF/UA)
- code 06 of codelist 196 (PDF/UA-2)
- code 10 of codelist 196 (No reading system accessibility options disabled (except))
- code 13 of codelist 196 (Single logical reading order)
- code 18 of codelist 196 (Accessible chem content as ChemML)
- code 22 of codelist 196 (Language tagging provided)
- code 28 of codelist 196 (Full alternative audio descriptions)
- code 30 of codelist 196 (ARIA roles provided)
- code 31 of codelist 196 (Accessible controls provided)
- code 32 of codelist 196 (Landmark navigation)
- code 38 of codelist 196 (Unusual words or abbreviations explained)
- code 40 of codelist 196 (Link purposes clear)
- code 95 of codelist 196 (Trusted intermediary’s web page for detailed accessibility information)
- code 96 of codelist 196 (Publisher’s web page for detailed accessibility information)
- code 97 of codelist 196 (Compatibility tested)
- code 98 of codelist 196 (Trusted Intermediary contact)
